fplus-lib/src/core/mod.rs

```diff
@@ -876,12 +882,29 @@ impl LDNApplication {
                         let signer = signers.0.get(1).unwrap();
                         let signer_address = signer.signing_address.clone();
                         let valid_notaries = Self::fetch_notaries().await?;
-                        if valid_notaries.is_valid(&signer_address) {
-                            dbg!("Valid notary");
-                            return Ok(true);
+                        if !valid_notaries.is_valid(&signer_address) {
```

Comment:
not sure why this was switched

Reply:
Because that's where the function ended with return ok(true) and outside it was return ok(false) and I didn't want to do the send comm to issue & add label inside the if considering all the other ifs were testing for false, not for true. So I just switched them - added return ok(false) inside, continued my code outside and changed at the end return Ok(true)